src/prepare_pcap.c

```diff
@@ -52,6 +66,14 @@ typedef struct _ipv6_hdr {
 } ipv6_hdr;
 
 
+#ifdef __DARWIN
+#define le16toh(x) OSSwapLittleToHostInt16(x)
+#endif
+
+#ifdef __HPUX
+#define le16toh(x) bswap_16(x)
+#endif
```

Comment:
This, along with the #ifdefs above should be moved to an endian.h. This is cluttering up this file.

(And some #if's should be replaced by #elif's.)